repose-aggregator/services/atom-feed/impl/src/main/resources/META-INF/schema/config/atom-feed-service.xsd

```diff
@@ -0,0 +1,189 @@
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+  -->
+
+<xs:schema elementFormDefault="qualified"
+           targetNamespace="http://docs.openrepose.org/repose/atom-feed-service/v1.0"
+           xmlns="http://docs.openrepose.org/repose/atom-feed-service/v1.0"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:html="http://www.w3.org/1999/xhtml"
+        >
+
+    <xs:element name="atom-feed-service" type="AtomFeedServiceConfigType"/>
+
+    <xs:complexType name="AtomFeedServiceConfigType">
+        <xs:sequence>
+            <xs:element name="feed" type="AtomFeedConfigType" minOccurs="1" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="AtomFeedConfigType">
+        <xs:all minOccurs="0">
+            <xs:element name="authentication" type="AuthenticationType">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            Defines the authentication mechanism to be used, and provides any necessary configuration.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:all>
+
+        <xs:attribute name="id" type="xs:string" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines a unique ID which may be used to identify this feed.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="uri" type="xs:anyURI" use="required">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines the URI at which this feed can be read.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="polling-frequency" type="PositiveIntType" use="optional" default="5">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines how often, in seconds, the Atom Feed service checks this feed for updates.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="entry-order" type="EntryOrderType" use="optional" default="update">
```

Comment:
Oh? Does "publish" or "read"?

This seems like a trade-off between verbosity and clarity. I'm pretty okay with "update", "publish", and "feed" orders. It's worth noting that the atom:entry fields which would be used are named "updated" and "published".